src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService REGISTRATION_SERVICE = new RegistrationServiceImpl();
+
+    @Test
+    void register_checkOnUserRegistration_isOk() {
+        User actual = new User();
+        actual.setLogin("Snowman");
+        actual.setPassword("123456");
+        actual.setAge(44);
+        User expected = REGISTRATION_SERVICE.register(actual);
+        assertEquals(expected, actual);
+    }
```

Comment:
Test methods should be independent. It seems that you are using the same `REGISTRATION_SERVICE` for all tests. As a result, the state of the `REGISTRATION_SERVICE` is shared among all tests, which is not a good practice. Consider using `@BeforeEach` annotation to initialize a new `RegistrationServiceImpl` for each test.